pvactools/tools/pvaccompare/server.py

```diff
@@ -0,0 +1,38 @@
+from http.server import SimpleHTTPRequestHandler, HTTPServer
+import sys
+
+
+class CORSRequestHandler(SimpleHTTPRequestHandler):
+    def end_headers(self):
+        self.send_header("Access-Control-Allow-Origin", "*")
+        self.send_header("Access-Control-Allow-Methods", "GET, OPTIONS")
+        self.send_header(
+            "Access-Control-Allow-Headers", "x-requested-with, content-type"
+        )
+        self.send_header("Cache-Control", "no-cache, no-store, must-revalidate")
+        self.send_header("Pragma", "no-cache")
+        self.send_header("Expires", "0")
+        super().end_headers()
+
+    def do_OPTIONS(self):
+        self.send_response(200)
+        self.end_headers()
+
+
+def main():
+    PORT = 8080
+    server = HTTPServer(("localhost", PORT), CORSRequestHandler)
+    print("Starting local server...")
+    print(
+        f"View reports at http://localhost:{PORT}/pvactools/tools/pvaccompare/html_report/main.html"
```

Comment:
This will only work if you happen to start the server from a directory that contains this file path underneath it, which wont' be the case once pVACtools is installed somewhere. You will need to get the actual absolute installation path of this file on the user's system or copy the main.html from its installation path to the working directory before starting the server (and then clean it up at KeyboardInterrupt).